src/Servers/Kestrel/Core/src/Internal/Http/HttpHeaders.Generated.cs

```diff
@@ -227,7 +227,14 @@ public override StringValues HeaderConnection
             }
             set
             {
-                _bits |= 0x2L;
+                if (!StringValues.IsNullOrEmpty(value))
+                {
+                    _bits |= 0x2L;
```

Comment:
Is there an enum somewhere that defines the meaning of these magic numbers?

Reply:
Oh never mind, I see

Reply:
They're auto-generated from here:
https://github.com/dotnet/aspnetcore/pull/36166/files#diff-aa5ab1fe02b00e816a5ec7b62df7ad54a91851ee89ff3decffa6ae614bf9829bR848